config/repos.yaml

```diff
@@ -11,6 +11,10 @@ theforeman/chef-handler-foreman:
   redmine: chef
 theforeman/community-templates:
   redmine: foreman
+  directory_labels:
+    job_templates: Remote execution
+    partition_tables_templates: Provisioning
+    provisioning_templates: Provisioning
```

Comment:
Is the key the directory name and the value the label? Does this handle nested directories?

Reply:
Yes to the first, no to the second: ```desired_labels = files.collect { |f| mapping[f.filename.split("/").first] }.reject { |l| l.nil? }.unique```

Reply:
Why the split on '/' ?

Reply:
Originally I just wanted the base directory because that's enough for community-templates. I suppose we could modify it to look for all directories

Reply:
I could already think of a katello case where a large chunk of UI is in 'engines/bastion_katello'

Reply:
It would also be great to look at top level files like `package.json`.

Reply:
@ehelms I'll try to modify it to support that in an efficient manner
@danseethaler as an unintended side-effect that currently works